File: Sample0001/Solution.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        String[] numbersStr = scanner.nextLine().split(" ");
        Integer[] numbers = new Integer[n];

        for (int i = 0; i < numbersStr.length; i++) {
            numbers[i] = Integer.parseInt(numbersStr[i]);
        }

        int sum = 0;
        for (int i = 0; i < numbers.length; i++) {
            sum += numbers[i];
        }

        System.out.println(sum);
    }
}

File: Sample0001/solution.cpp
#include <iostream>

int main () {
    int n;
    int sum = 0;

    std::cin >> n;

    int numbers[n];
    for (int i = 0; i < n; i++) {
        std::cin >> numbers[i];
    }

    for (int i = 0; i < n; i++) {
        sum += numbers[i];
    }

    std::cout << sum << std::endl;
    return 0;
}